nucliadb_ingest/nucliadb_ingest/orm/resource.py

```diff
@@ -263,27 +263,70 @@ async def generate_broker_message(self) -> BrokerMessage:
 
         fields = await self.get_fields(force=True)
         for ((type_id, field_id), field) in fields.items():
-            extracted_text_cf = await field.get_extracted_text_cf()
-            if extracted_text_cf is not None:
-                etw = ExtractedTextWrapper()
-                etw.file.CopyFrom(extracted_text_cf)
-                bm.extracted_text.append(etw)
+            # Value
+            if type_id == FieldType.TEXT:
+                value = await field.get_value()
+                bm.texts[field_id].CopyFrom(value)
+            elif type_id == FieldType.LINK:
+                value = await field.get_value()
+                bm.links[field_id].CopyFrom(value)
+            elif type_id == FieldType.FILE:
+                value = await field.get_value()
+                bm.files[field_id].CopyFrom(value)
+            elif type_id == FieldType.CONVERSATION:
+                value = await field.get_value()
+                bm.conversations[field_id].CopyFrom(value)
+            elif type_id == FieldType.KEYWORDSET:
+                value = await field.get_value()
+                bm.keywordsets[field_id].CopyFrom(value)
+            elif type_id == FieldType.DATETIME:
+                value = await field.get_value()
+                bm.datetimes[field_id].CopyFrom(value)
+            elif type_id == FieldType.LAYOUT:
+                value = await field.get_value()
+                bm.layouts[field_id].CopyFrom(value)
```

Comment:
What if the type is non of these? Do we need a `else: raise ValueError()` branch at the end?

Reply:
in case there is else we do not want to raise, we do not want to export.